docs/get_started/get_started_demos.md

```diff
@@ -90,29 +96,26 @@ This guide uses the OpenVINO™ Model Downloader to get pre-trained models. You
 
    .. code-block:: sh
 
-      cd /opt/intel/openvino_2021/deployment_tools/tools/model_downloader/
-      python3 info_dumper.py --print_all
+      info_dumper --print_all
 
 .. tab:: Windows
 
    .. code-block:: bat
 
-      cd <INSTALL_DIR>\deployment_tools\tools\model_downloader\
-      python info_dumper.py --print_all
+      info_dumper --print_all
 
 .. tab:: macOS
 
    .. code-block:: sh
 
-      cd /opt/intel/openvino_2021/deployment_tools/tools/model_downloader/
-      python3 info_dumper.py --print_all
+      info_dumper --print_all
 
 @endsphinxdirective
```

Comment:
BTW, do we need tabs for OSes? I suppose python tools work in unified manner for al operation systems and we can provide a single command.

Reply:
the same is valid for other places as well, which are OS-independent.